man/systemd-gpt-auto-generator.xml

```diff
@@ -32,22 +32,18 @@
   <refsect1>
     <title>Description</title>
 
-    <para><filename>systemd-gpt-auto-generator</filename> is a unit
-    generator that automatically discovers root,
-    <filename>/home</filename>, <filename>/srv</filename> and swap
-    partitions and creates mount and swap units for them, based on the
-    partition type GUIDs of GUID partition tables (GPT),
-    see <ulink url="http://www.uefi.org/specifications">UEFI Specification</ulink>, chapter 5.
-    It implements the <ulink
-    url="https://www.freedesktop.org/wiki/Specifications/DiscoverablePartitionsSpec/">Discoverable
-    Partitions Specification</ulink>. Note that this generator has no
-    effect on non-GPT systems, or where the directories under the
-    mount points are already non-empty. Also, on systems where the
-    units are explicitly configured (for example, listed in
-    <citerefentry
-    project='man-pages'><refentrytitle>fstab</refentrytitle><manvolnum>5</manvolnum></citerefentry>),
-    the units this generator creates are overridden, but additional
-    implicit dependencies might be created.</para>
+    <para><filename>systemd-gpt-auto-generator</filename> is a unit generator that automatically discovers
+    root, <filename>/home/</filename>, <filename>/srv/</filename>, the EFI System Partition, the Extended
+    Boot Loader Partition and swap partitions and creates mount and swap units for them, based on the
+    partition type GUIDs of GUID partition tables (GPT), see <ulink
+    url="http://www.uefi.org/specifications">UEFI Specification</ulink>, chapter 5.  It implements the <ulink
+    url="https://www.freedesktop.org/wiki/Specifications/DiscoverablePartitionsSpec/">Discoverable Partitions
+    Specification</ulink>. Note that this generator has no effect on non-GPT systems, or where the
+    directories under the mount points are already non-empty. Also, on systems where the units are explicitly
```

Comment:
Stuff under `/efi` is ignored, so this sentence is wrong? (Or was it only in dissect-mount?)

Reply:
no we do check whether /efi is already populated and if so don't over-mount it. but the text is still a bit skewed, since the generator has effect on other dirs even if some dir is already populated